.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - 'node'
+  - '6'
+  - '4'
+addons:
+  code_climate:
+    repo_token: 'your repo token'
+notifications:
+  slack: feathersjs:T4tvpJFt27wJZABjLSg5OqMv
```

Comment:
Does this need to be encrypted? Why is this still gettting included? cc/ @feathersjs/core-team